internal/proto/opamp.proto

```diff
@@ -111,6 +111,11 @@ message ServerToAgent {
     // be omitted in subsequent ServerToAgent messages by setting it to
     // UnspecifiedServerCapability value.
     ServerCapabilities capabilities = 8;
+
+    // Allows the Server to instruct the agent to perform a command, e.g. RESTART. This field should not be specified
+    // with fields other than instance_uid and capabilities. If specified, other fields will be ignored and the command
+    // will be performed.
+    ServerToAgentCommand command = 10;
```

Comment:
@pmm-sumo please also add the field 9 `agent_identification` in a new PR. I was confused by the gap.